alica-plan-designer-fx-codegeneration/src/main/java/de/unikassel/vs/alica/generator/cpp/XtendTemplates.xtend

```diff
@@ -1011,15 +1011,21 @@ namespace alica
                 «ENDIF»
                 /*PROTECTED REGION END*/
             protected:
+                /*PROTECTED REGION ID(override«plan.id») ENABLED START*/
```

Comment:
Instead of a new protected region, just generated them in the protected region below, but starting out commented